sdk/purview/azure-analytics-purview-workflow/src/samples/java/com/azure/analytics/purview/workflow/ReadmeSamples.java

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.analytics.purview.workflow;
+
+public final class ReadmeSamples {
+    public void readmeSamples() {
+        // BEGIN: com.azure.analytics.purview.workflow.readme
+        // END: com.azure.analytics.purview.workflow.readme
```

Comment:
The generator only provides an empty samples class that needs to be manually updated to add samples.

Reply:
Fixed, thanks